src/phoenix/trace/llama_index/callback.py

```diff
@@ -198,6 +208,27 @@ def on_event_start(
         **kwargs: Any,
     ) -> CBEventID:
         event_id = event_id or str(uuid4())
+        if (
+            event_type is CBEventType.EXCEPTION
+            and payload
+            and (error := payload.get(EventPayload.EXCEPTION))
```

Comment:
what if these two are not true, wouldn't the exception fall through the `if` statement?

Reply:
addressed in [7489389](https://github.com/Arize-ai/phoenix/pull/1449/commits/74893897ac3a8c149cde28d90900ec09ea0e9997)